### PR TITLE
Fixes sticky tape spam

### LINKED ISSUE #52673: If you use sticky tape multiple times very fast on an item, it will be wrapped multiple times
## Round ID:
[143356](https://scrubby.melonmesa.com/round/143356)

## Reproduction:

1) Click on something a bunch of times with sticky tape without waiting
2) Wait for all of the progress bars to finish
3) See that the object is wrapped, like, 5 million times

The reason I'm pretty sure this is a bug is because you actually aren't allowed to wrap something in sticky tape if its already finished being wrapped. However, if you just start wrapping it a bunch and wait for them all to finish, it never checks if its already covered. 

image

don't fix this its funny

### PR DESCRIPTION
**Pull request by Fhaxaris**
fixes: #52673

### CODE REVIEW

**Comment by Doosteroni** [comment]
noooo it's funny

File: code/game/objects/items/stacks/tape.dm [hunk 1]
```diff
@@ -27,6 +27,9 @@
 		to_chat(user, "<span class='warning'>[I] is already coated in [src]!</span>")
 		return
 
+	if(INTERACTING_WITH(user, I))
```
Comment (optimumtact) [inline code/game/objects/items/stacks/tape.dm:30]: don't do this, check in the do_after and just after the do_after to see if `I.embedding && I.embedding == conferred_embed` is set and bail out of the do after.

Comment (Fhaxaris) [reply code/game/objects/items/stacks/tape.dm]: ok but why, is it more foolproof in case more than one person is doing something or what.

Comment (optimumtact) [reply code/game/objects/items/stacks/tape.dm]: just cleaner than checking if the user is already interacting with the item

**Pull request merged**